commons/get-help/faq.md

```diff
@@ -135,4 +135,12 @@ Clever Cloud does not give you access to a server or a VPS, it makes your applic
 
 ## I want to user Clever Cloud on my own premises, is that possible?
 
-Yes, since 2016 Clever Cloud is packaged for private datacenter. This offer called "Clever Cloud On Premises" is avaialble upon request: you can send a mail to <sales@clever-cloud.com> or visit [https://www.clever-cloud.com/on-premises](www.clever-cloud.com/on-premises) for more infos.
\ No newline at end of file
+Yes, since 2016 Clever Cloud is packaged for private datacenter. This offer called "Clever Cloud On Premises" is avaialble upon request: you can send a mail to <sales@clever-cloud.com> or visit [https://www.clever-cloud.com/on-premises](www.clever-cloud.com/on-premises) for more infos.
+
+## Where are my applications and databases located?
```

Comment:
s/databases/add-ons/ ? You talk about add-on at the end of the paragraph

Reply:
Fixed :)